src/main/msp/msp.c

```diff
@@ -3646,7 +3646,7 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
         int32_t ned_vel_east = (int32_t)sbufReadU32(src);   // ned_vel_east
         gpsSol.groundSpeed = (uint16_t)sqrtf((ned_vel_north * ned_vel_north) + (ned_vel_east * ned_vel_east));
         (void)sbufReadU32(src);             // ned_vel_down
-        gpsSol.groundCourse = ((uint16_t)sbufReadU16(src) % 360);   // ground_course
+        gpsSol.groundCourse = ((uint16_t)sbufReadU16(src) % 360) * 10;   // ground_course
```

Comment:
We can also do this for more resolution, afaik.
```suggestion
        gpsSol.groundCourse = ((uint16_t)sbufReadU16(src) % 3600);   // ground_course
```